project/src/main/world/CareerMap.tscn

```diff
@@ -279,6 +279,7 @@ modulate = Color( 1, 1, 1, 0.501961 )
 margin_right = 1024.0
 margin_bottom = 600.0
 rect_min_size = Vector2( 1024, 600 )
+mouse_filter = 2
```

Comment:
Mention this in the commit comment

Reply:
Expanded commit comment; `Changed ScrewportTexRect mouse_filter to 'ignore' to prevent it from
stealing mouse events in menus.`